financeager/cli.py

```diff
@@ -35,7 +35,7 @@ def main():
     sys.exit(run(**_parse_command()))
 
 
-def run(command=None, config=None, verbose=False, **cl_kwargs):
+def run(command=None, config_path=None, verbose=False, **cl_kwargs):
```

Comment:
Can you name it `config_filepath`? That's the name of the variable that will be automatically generated by argparse from the `--config-filepath` option.

Reply:
Can you update the mention of 'config' in the docstring?

Reply:
> Can you name it `config_filepath`? That's the name of the variable that will be automatically generated by argparse from the `--config-filepath` option.

And delete this line? config_filepath = config_path

Reply:
Yes.